src/object/cli_obj.c

```diff
@@ -2627,3 +2662,369 @@ dc_obj_query_key(tse_task_t *api_task)
 
 	return rc;
 }
+
+struct obj_verify_one_rdg_unpack_arg {
+	daos_handle_t			 oh;
+	daos_obj_id_t			 oid;
+	uint32_t			 shard;
+	daos_epoch_t			*epoch;
+	struct obj_enum_dkeys_arg	*arg;
+};
+
+static int
+dc_obj_fetch_sepc_wrap(daos_handle_t oh, daos_handle_t th, unsigned int flags,
+		       unsigned int shard, daos_key_t *dkey, unsigned int nr,
+		       daos_iod_t *iods, d_sg_list_t *sgls)
+{
+	tse_task_t	*task;
+	int		 rc;
+
+	rc = dc_obj_fetch_task_create(oh, th, flags, shard, dkey, nr, iods,
+				      sgls, NULL, NULL, NULL, &task);
+	if (rc == 0)
+		rc = dc_task_schedule(task, true);
+
+	return rc;
+}
+
+static int
+dc_obj_verify_one_rdg_unpack_cb(struct daos_enum_unpack_io *io, void *arg)
```

Comment:
this function is specific for replication, and there is no match in EC even in the future, so I think we should create a new file, e.g. src/object/cli_replica.c, and move this function, and probably some others functions to this file.
Again, EC is underway and I think we should keep this in mind all the time.